playwright-e2e/dsm/pages/participant-list-page.ts

```diff
@@ -263,14 +264,34 @@ export default class ParticipantListPage extends DsmPageBase {
 
       if (rgpProbandTab === true) {
         //If the participant has participantData, this seems to mean there's a proband tab in the account
-        const participantData = responseJson.participants[index].participantData;
+        const participantData = responseJson.participants[index].participantData[0];
         firstName = responseJson.participants[index].esData.profile.firstName;
         if (participantData && (firstName === testParticipantFirstName)) {
           shortID = responseJson.participants[index].esData.profile.hruid;
           logInfo(`Found the RGP participant ${shortID} to have a proband tab`);
           break;
         }
       }
+
+      //Go to the next page if none of the participants in the current page are relevant
```

Comment:
planning to move this to be a general method in a separate PR since it ended up being used twice